app/assets/stylesheets/layout.css.scss

```diff
@@ -32,56 +32,62 @@ a {
   color: #537344;
 }
 
-.container {
-  width: 100%;
-  padding-right: 0px;
-  padding-left: 0px;
-  background-color: white;
-  padding-bottom: 150px;
-}
-
-.alert-danger {
-  background-image: linear-gradient(to bottom,#f2dede 0,#e7c3c3 100%);
-  background-repeat: repeat-x;
-  border-color: #dca7a7;
-  margin-bottom: 20px;
-  box-shadow: inset 0 1px 0 rgba(255,255,255,.25),0 1px 2px rgba(0,0,0,.05);
-  margin-bottom: 20px;
-  border-radius: 4px;
-  padding-left: 15px;
-}
-
 //Header
 
 #header-box {
   background-image: image-url('airliftlogo.jpg');
   background-repeat: no-repeat;
-  background-position-x: 40px;
+  background-position-x: 20px;
   background-size: 100px;
   background-color: white;
   width: 100%;
-  height: 170px;
-  border-bottom: 4px solid #ccc;
-}
-
-#restarant_name h1 {
-  font-size: 60px;
+  // height: 170px;
 }
 
 #restaurant_name a {
   text-decoration: none;
   color: black;
-  font-size: 50px;
+  font-size: 40px;
+}
+
+#restaurant_name h4 {
+  margin-bottom: 15px;
+}
+
+.btn.btn-link.btn-default.register {
+  margin-right: -30px;
+}
+
+.logout {
+  margin-top: 2px;
+}
+
+.search-nav {
+  background-color: #D7D8D9;
+  margin: 0;
+  padding: 7px;
 }
 
 #search-bar {
-  float: left;
-  margin-top: 10px;
-  padding-left: 69px;
+  margin-left: -21px;
 }
 
-#user-nav {
-  text-align: right;
+.btn-default.view-items {
+  height: 35px;
+  padding-top: 7px;
```

Comment:
have you considered a padding-top of 6px or even 8px?  Have you really considered all the alternatives to 7px?